Sources/HipMobileUI/App.xaml

```diff
@@ -36,13 +36,135 @@
             <Color x:Key="LightGreenBarColor">#A9D839</Color>
             <Color x:Key="YellowBarColor">#FDDB37</Color>
             <Color x:Key="OrangeBarColor">#FEB533</Color>
-            <Color x:Key="RedBarColor">#FF8E59</Color>       
+            <Color x:Key="RedBarColor">#F22F0C</Color>       
 
             <Style TargetType="Label">
                 <Setter Property="FontSize" Value="12" />
                 <Setter Property="TextColor" Value="Black" />
             </Style>
 
+            <Style x:Key="Ssp-Black" TargetType="Label">
+                <Setter Property="FontFamily">
+                    <Setter.Value>
+                        <OnPlatform x:TypeArguments="x:String">
+                            <OnPlatform.Android>SourceSansPro-Black.ttf#SourceSansPro-Black</OnPlatform.Android>
```

Comment:
Instead of `<OnPlatform.Android>` and `<OnPlatform.iOS>`
you should use `<On Platform="Android">` and `<On Platform="iOS">`.